apps/website/app/ui/sections/ButtonsSection.tsx

```diff
@@ -0,0 +1,187 @@
+): Array<ButtonListProps> => [
+  { children },
+  { children, variant: "secondary" },
+  { children, variant: "light" },
+  { children, variant: "tertiary" },
+  { children, variant: "ghost" },
+  { children, active: true },
+  { children, disabled: true },
+  { children, variant: "ghost", disabled: true },
+];
+
+const getMainBtnCombos = (
+  children: string = "Button"
+): Array<ButtonListProps> => [
+  ...getBaseBtnCombos(children),
+  { children, variant: "neutral" },
+];
+
+type ExtendedButtonProps = Omit<ButtonListProps, "children">;
+type ExtendedButtonLinkProps = Omit<ButtonLinkListProps, "children">;
+
+const extendBtnCombos = (
+  btnPropsList: Array<ButtonListProps>,
+  newProp: ExtendedButtonProps | ExtendedButtonLinkProps
+): Array<ButtonListProps> =>
+  btnPropsList.map(buttonProps => ({
+    ...buttonProps,
+    ...newProp,
+  }));
+
+const regularBtnCombos: Array<ButtonListProps | ButtonLinkListProps> =
+  getMainBtnCombos();
+const successBtnCombos: Array<ButtonListProps | ButtonLinkListProps> =
+  extendBtnCombos(getBaseBtnCombos(), { colorScheme: "success" });
+const dangerBtnCombos: Array<ButtonListProps | ButtonLinkListProps> =
+  extendBtnCombos(getBaseBtnCombos(), { colorScheme: "danger" });
+
+const buttonsList: {
+  headers: Array<string>;
+  comboNames: Array<string>;
+  combos: Array<Array<ButtonListProps | ButtonLinkListProps>>;
+} = {
+  headers: [
+    "primary",
+    "secondary",
+    "light",
+    "tertiary",
+    "Ghost",
+    "Active",
+    "Disabled",
+    "Disabled 👻",
+    "neutral",
+  ],
+  comboNames: ["main", "success", "danger"],
+  combos: [regularBtnCombos, successBtnCombos, dangerBtnCombos],
+};
+
+export const ButtonSection = () => (
+  <ButtonsSection component={Button} btnList={buttonsList}>
+    Buttons
+  </ButtonsSection>
+);
+
+export const ButtonLinkSection = () => (
+  <ButtonsSection component={ButtonLink} btnList={buttonsList}>
+    ButtonLinks
+  </ButtonsSection>
+);
+
+const ButtonsSection = ({
+  children,
+  btnList,
+  component,
+}: ButtonSectionProps) => {
+  const Component = component;
+  const [size, setSize] = useState<ButtonSize>("md");
+  const [width, setWidth] = useState<ButtonWidth>("normal");
+
+  return (
+    <Section title={children?.toString()}>
+      <div className="flex mb-4 space-x-3">
+        <SelectProp
+          name="btnSize"
+          title="size"
+          value={size}
+          set={setSize}
+          options={[
+            { value: "xs", label: "xs" },
+            { value: "sm", label: "sm" },
+            { value: "md", label: "md" },
+            { value: "lg", label: "lg" },
+            { value: "xl", label: "xl" },
+          ]}
+        />
+        <SelectProp
+          name="btnWitdh"
+          title="width"
+          value={width}
+          set={setWidth}
+          options={[
+            { value: "normal", label: "normal" },
+            { value: "fit", label: "fit" },
+            { value: "full", label: "full" },
+          ]}
```

Comment:
Maybe we can put this options in an array?

Reply:
I think it's okay